extensions/azurecore/src/azureResource/providers/databaseServer/databaseServerTreeDataProvider.ts

```diff
@@ -54,8 +55,24 @@ export class AzureResourceDatabaseServerTreeDataProvider implements azureResourc
 					dark: this._extensionContext.asAbsolutePath('resources/dark/sql_server_inverse.svg'),
 					light: this._extensionContext.asAbsolutePath('resources/light/sql_server.svg')
 				},
-				collapsibleState: TreeItemCollapsibleState.None,
-				contextValue: AzureResourceItemType.databaseServer
+				collapsibleState: process.env.NODE_ENV === 'development' ? TreeItemCollapsibleState.Collapsed : TreeItemCollapsibleState.None,
+				contextValue: AzureResourceItemType.databaseServer,
+				payload: {
+					id: generateGuid(),
+					connectionName: undefined,
+					serverName: databaseServer.fullName,
+					databaseName: databaseServer.defaultDatabaseName,
+					userName: databaseServer.loginName,
+					password: '',
+					authenticationType: 'SqlLogin',
+					savePassword: true,
+					groupFullName: '',
+					groupId: '',
+					providerName: 'MSSQL',
```

Comment:
can we assume the provider will be `MSSQL`?

Reply:
I'm not sure, the extension was already assuming MSSQL when it was opening the connection dialog. I'm assuming you could tell from the resource type. But I didn't look too much into the extension.